modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -1,68 +1,62 @@
+To access the internal URL, you must specify the authorization heads as part of the request URL. To test an API without supplying authorization headers, enable the external  URL for this project.
 ====
 
-== Procedure
-
-* If you are using the text editor in API Designer:
-+
-. Turn on the mocking service by clicking the *Mocking service* slider in the top-right corner of the editor.
-. In the API summary panel, expand completely the resource with the endpoint that you want to test.
+. From API Designer, open the project containing the API you want to test.
++ 
+You can use either the text editor or the visual editor.
+. Select the root file of your project.
+. Click the *Documentation* icon in the right-hand tool bar.
+. From the drop-down menu, select *Summary*.
+. If you have defined `baseURL` in your API specification:
++ 
+* Click the *Select server* dropdown and select one of the following:
 +
-_Result:_ After you expand the resource, you will see one or more buttons. The labels on the buttons match the different request types for the resource.
-. Click the button that matches the request type that you want to simulate.
-+
-_Result:_ The panel displays information about the request type, its parameters, and the codes and messages that it returns.
-. Click *Try It* in the top-right corner of the editor.
-+
-_Result:_ The URI for the endpoint is displayed. If the endpoint requires parameters, specify values for them. Also, you can view the headers and add custom headers.
-. Click *Send*.
-+
-_Result:_ The panel displays the result below the *Send* button.
+** baseUri: uses the baseUri as defined in your API specification
+** Mocking Service: uses the API mocking service 
 
-* If you are using the visual editor in API Designer:
-+
-. Turn on the mocking service by clicking the *Mocking service* slider in the top-right corner of the editor.
-. In the left panel, select the resource with the endpoint that you want to test.
-. At the top of the center panel of the editor, select the request type that you want to test.
-. Click *Try It* in the lower-right corner of the editor.
+. Click one of the endpoints, then click *Try it*.
+. Click *Send.*
 +
-_Result:_ The right panel displays the request URL. It also requires you to specify the values for any parameters that the endpoint requires. Finally, it displays headers and allows you to add custom headers.
-. Click *Send*.
-+
-_Result:_ The panel displays the result below the *Send* button.
+API Designer displays the HTTP return code as specified in your API. Alternately, you can test the API by copying the *Request URL* and past it into a web browser or Postman.
+
+[IMPORTANT]
+====
+To select the default option that appears in the *Select server* drop down, click the *Select by Default* slider under *Mocking Service Configuration*.
```

Comment:
Just to be clear that the switch only applies to Mocking Service, I would say `To select Mocking Service as the default option in *Select server* drop down....`

Reply:
Fixed typo.

Revised to: To select Mocking Service as the default option in the *Select server* drop-down, click the *Select by Default* slider under *Mocking Service Configuration*.

Reply:
LGTM!